packages/grid/test/lit-renderer-directives.test.js

```diff
@@ -6,7 +6,7 @@ import { html, render } from 'lit';
 import { columnBodyRenderer, columnFooterRenderer, columnHeaderRenderer, gridRowDetailsRenderer } from '../lit.js';
 import { getCellContent, getContainerCell } from './helpers.js';
 
-async function renderGrid(container, { items, header, footer, rowDetails }) {
+function renderGrid(container, { items, header, footer, rowDetails }) {
```

Comment:
This change seems to break tests:

```
packages/grid/test/lit-renderer-directives.test.js:
 ❌ lit renderer directives > gridRowDetailsRenderer > arguments > should pass the item to the renderer
TypeError: Cannot read properties of null (reading 'args')
  at o.<anonymous> (packages/grid/test/lit-renderer-directives.test.js:78:38)
```

Reply:
Ok, there was a couple of missing `nextFrame()`. Apparently, `async` functions can be executed asynchronously regardless of the `await` presence inside. This is a surprise for me because  I always thought that `async` functions are only executed asynchronously if they have at least one `await`. Anyway, thanks to this rule, we can avoid such situations from now on.